### DISCUSSION TITLE
ideavim IDE differences between PyCharm and Rider

### DISCUSSION THREAD

**Discussion by psarras** [JetBrains/ideavim#748] [Q&A] [answered]
Hi, I want a shortcut to run tests for the current method.

For Rider, this is `RiderUnitTestRunContextSplitAction` for PyCharm is simply `RunClass`

```
map <leader>tt <action>(RiderUnitTestRunContextSplitAction)
map <leader>tt <action>(RunClass)
```

I would ideally like to have one shortcut that runs the tests, independent on which IDE I am using.

**Comment by lippfi** [comment, marked as answer]
Hi! Please check out our documentation about the `ide` option and IDE-dependent configurations
https://github.com/JetBrains/ideavim#ide-specific-options

```
if &ide =~? 'rider'
    map <leader>tt <action>(RiderUnitTestRunContextSplitAction)
else
    map <leader>tt <action>(RunClass)
endif
```
